fastlane_bot/helpers/wrap_unwrap_processor.py

```diff
@@ -0,0 +1,344 @@
+from typing import Any, List, Dict
+
+from fastlane_bot.helpers import TradeInstruction, RouteStruct
+from dataclasses import asdict
+
+
+class WrapUnwrapProcessor:
+    """
+    Handles the decision-making for wrapping & unwrapping gas tokens within trade routes.
+
+    This class tracks token balances of trades, and decides when to add wrap or unwrap trades.
+
+    Typical usage example:
+    wrap_unwrap_processor = WrapUnwrapProcessor(cfg=self.ConfigObj)
+    route_struct_processed = wrap_unwrap_processor.add_wrap_or_unwrap_trades_to_route(trade_instructions=split_calculated_trade_instructions, route_struct=route_struct, flashloan_struct=flashloan_struct)
+
+    """
+
+    def __init__(self, cfg: Any):
+        self.cfg = cfg
+
+    def add_wrap_or_unwrap_trades_to_route(
+        self,
+        trade_instructions: List[TradeInstruction],
+        route_struct: List[Dict],
+        flashloan_struct: List,
+    ) -> List[Dict]:
```

Comment:
In file routerhandler.py, [there is a function](https://github.com/bancorprotocol/fastlane-bot/blob/381-add-ability-to-split-native-vs-wrapped-carbon-orders/fastlane_bot/helpers/routehandler.py#L354-L356) with the same name, which takes the same input arguments in the same order, and returns the same output type.
The implementation of that function looks suspiciously similar to the implementation of this function.
Consider implementing the necessary functionality in a single place in the code.

Reply:
Removed the now-unused code from Routehandler.